DuckDuckGo/AutofillLoginSettingsListViewController.swift

```diff
@@ -552,6 +552,21 @@ extension AutofillLoginSettingsListViewController: UITableViewDelegate {
             return 0
         }
     }
+
+    func tableView(_ tableView: UITableView, willDisplayHeaderView view: UIView, forSection: Int) {
+        if let view = view as? UITableViewHeaderFooterView {
+            let theme = ThemeManager.shared.currentTheme
+            view.textLabel?.textColor = theme.tableHeaderTextColor
```

Comment:
It would be nice to be able to move away from needing the theme thing, but I don't have a good sense of what's holding us back there. Is there anything that necessitates it versus just being a lot of work for us to get rid of it?

Reply:
I think once we have migrated all the theme colours to the design system we could discuss dropping the theme entirely.  The only advantage of keeping it is if we want to do custom themes, but even then because we've been using the design system colours directly in places it'd be a lot of work.